projects/rates_example_test.go

```diff
@@ -0,0 +1,324 @@
+package projects_test
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/teamwork/twapi-go-sdk"
+	"github.com/teamwork/twapi-go-sdk/projects"
+	"github.com/teamwork/twapi-go-sdk/session"
+)
+
+func ExampleRateUserGet() {
+	engine := twapi.NewEngine(session.NewBearerToken("your_token", "https://your-domain.teamwork.com"))
+
+	req := projects.NewRateUserGetRequest(12345) // User ID
+	// Configure optional filters
+	req.Filters.IncludeUserCost = true
+	req.Filters.Include = []string{"projects", "currencies"}
+
+	resp, err := projects.RateUserGet(context.Background(), engine, req)
+	if err != nil {
+		fmt.Printf("Error: %s\n", err)
+		return
+	}
+
+	if resp.InstallationRate != nil {
+		fmt.Printf("User installation rate: %d\n", *resp.InstallationRate)
+	}
+	fmt.Printf("Number of project rates: %d\n", len(resp.ProjectRates))
+	fmt.Printf("Number of multi-currency rates: %d\n", len(resp.InstallationRates))
+
+	if resp.UserCost != nil {
+		fmt.Printf("User cost: %d\n", *resp.UserCost)
+	}
+}
+
+func ExampleRateInstallationUserList() {
+	engine := twapi.NewEngine(session.NewBearerToken("your_token", "https://your-domain.teamwork.com"))
+
+	req := projects.NewRateInstallationUserListRequest()
+	req.Filters.PageSize = 10 // Get first 10 users
+
+	resp, err := projects.RateInstallationUserList(context.Background(), engine, req)
+	if err != nil {
+		fmt.Printf("Error: %s\n", err)
+		return
+	}
+
+	fmt.Printf("Found %d user rates\n", len(resp.UserRates))
+
+	// Iterate through all pages
+	for {
+		for _, userRate := range resp.UserRates {
+			fmt.Printf("User %d has rate %d\n", userRate.User.ID, userRate.Rate)
+		}
+
+		// Get next page if available
+		nextReq := resp.Iterate()
+		if nextReq == nil {
+			break
+		}
+
+		resp, err = projects.RateInstallationUserList(context.Background(), engine, *nextReq)
+		if err != nil {
+			fmt.Printf("Error: %s\n", err)
+			return
+		}
+	}
```

Comment:
For the pagination example, we could follow the iterator pattern:

```go
next, err := twapi.Iterate[projects.RateInstallationUserListRequest, *projects.RateInstallationUserListResponse](
  context.Background(),
  engine,
  projects.NewRateInstallationUserListRequest(),
)
if err != nil { ... }

for {
  response, hasNext, err := next()
  if err != nil { ... }
  if response == nil {
    break
  }
  for _, userRate := range response.UserRates {
    fmt.Printf("User %d has rate %d\n", userRate.User.ID, userRate.Rate)
  }
}
```

This would follow the pattern from [this example](https://github.com/Teamwork/twapi-go-sdk/blob/2ff08ba1a6f1e0bcdac13b2d50b14145b78065b3/examples/iterator/iterator.go).